Home.html

```diff
@@ -14,64 +14,98 @@
     </head>
     <body>
         <main>
-            <section class="home p-2">
+            <section class="home p-2" id="home-section">
                 <nav class="navbar navbar-expand-sm">
-                    <div class="container-fluid">
+                    <div class="container-fluid hamburger-container">
                         <img class="hamburger-btn" src="assets/menu-icon-png-3-lines-10.jpg" alt="Hamburger menu">
                     </div>
                 </nav>
                     <h5>"Hello Travelers"</h5>
-                    <h1>Afghan<br>Travel Agency</h1>
-                    <div class="description container-fluid">
+                    <h1>Afghan Travel<br>Agency</h1>
+                    <div class="ps-4 pe-4 description container-fluid">
                     <p>We make sure you travel with ease of mind<br>If you need help with any challenges of traveling including getting a passport, visa, plane ticket, bank statement and etc... you're in the right place</p>
                     </div>
-                    <h4 class="date">Current date</h4>
+                    <h4 class="date"></h4>
             </section>
+            <ul class="nav-links">
+                <li><img src="assets/twitter.png" alt="twitter"></li>
+                <li><img src="assets/facebook-app-symbol.png" alt="facebook"></li>
+                <li>English</li>
+                <li>My Page</li>
+                <li>Logout</li>
+            </ul>
             <ul class="nav-items">
                 <img src="assets/close.png" alt="cross" class="close-cross">
-                <li><a href="About.html">About</a></li>
+                    <div class="logo-container d-inline">
+                        <a href="Home.html"><img src="assets/travel.png" alt="Logo" class="d-inline nav-logo"></a>
+                        <p class="d-inline">ATA</p>
+                    </div>
+                <li class="li-1"><a href="About.html">About</a></li>
                 <li><a href="">Program</a></li>
                 <li><a href="">Join</a></li>
                 <li><a href="">Sponser</a></li>
```

Comment:
- [ ] To make syour page looks more professional, I suggest you correct the above typo error. You wrote `Sponser` instead of `Sponsor`